Atomic/atomic.py

```diff
@@ -645,36 +645,62 @@ def gen_cmd(self, cargs):
             args.append(c)
         return " ".join(args)
 
+    def get_fq_name(self, image_info):
+        if len(image_info['RepoTags']) > 1:
+            raise ValueError("\n{} is tagged with multiple repositories. "
+                             "Please use a repository name instead.\n".format(self.image))
+        else:
```

Comment:
I think the handling of the list and its index is fine. If you are worried, I can set a condition for that.   get_fq_name returns the repotag so it can be used as a fully qualified name to look up things remotely.